File: DotNET/Endpoint Examples/JSON Payload/delete-resource.cs
var client = new HttpClient();

// By default, we use the US-based API service. This is the primary endpoint for global use.
var pdfrestURL="https://api.pdfrest.com";

// For GDPR compliance and enhanced performance for European users, you can switch to the EU-based service by uncommenting the URL below.
// For more information visit https://pdfrest.com/pricing#how-do-eu-gdpr-api-calls-work
//var pdfrestURL="https://eu-api.pdfrest.com";

var request = new HttpRequestMessage(HttpMethod.Delete, $"{pdfrestURL}/resource/xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx");
request.Headers.Add("api-key", "xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx");
var response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
Console.WriteLine(await response.Content.ReadAsStringAsync());

File: JavaScript/Endpoint Examples/JSON Payload/delete-resource.js
const axios = require("axios");

// By default, we use the US-based API service. This is the primary endpoint for global use.
let api_url = "https://api.pdfrest.com"

// For GDPR compliance and enhanced performance for European users, you can switch to the EU-based service by uncommenting the URL below.
// For more information visit https://pdfrest.com/pricing#how-do-eu-gdpr-api-calls-work
//let api_url = "https://eu-api.pdfrest.com"

let config = {
  method: "delete",
  maxBodyLength: Infinity,
  url: api_url + "/resource/xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
  headers: {
    "api-key": "xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
  },
};

axios
  .request(config)
  .then((response) => {
    console.log(JSON.stringify(response.data));
  })
  .catch((error) => {
    console.log(error);
  });
